package.json

```diff
@@ -39,12 +39,7 @@
 		"pjv": "./bin/pjv"
 	},
 	"files": [
-		"bin/",
-		"demo/",
-		"index.html",
-		"LICENSE.md",
-		"package.json",
-		"README.md"
+		"bin/"
```

Comment:
Noting because this always surprises me: per https://docs.npmjs.com/cli/configuring-npm/package-json#files, files in the `"bin"` and `"main"` field are always included. That's why only `bin/` is needed here.